src/supla/network/esp32eth.h

```diff
@@ -86,31 +86,28 @@ class ESPETH : public Supla::Network {
   void setup() override {
     WiFiEventId_t event_gotIP = WiFi.onEvent(
         [](WiFiEvent_t event, WiFiEventInfo_t info) {
-          Serial.print(F("local IP: "));
-          Serial.println(ETH.localIP());
-          Serial.print(F("subnetMask: "));
-          Serial.println(ETH.subnetMask());
-          Serial.print(F("gatewayIP: "));
-          Serial.println(ETH.gatewayIP());
-          Serial.print(F("ETH MAC: "));
-          Serial.println(ETH.macAddress());
+          SUPLA_LOG_INFO("local IP: ", ETH.localIP());
```

Comment:
Those SUPLA_LOG_ macros are not a simple drop in replacement for Serial.print methods from Arduino.

SUPLA_LOG_ is a wrapper to printf style logging, so first you provide string in quotes with places marked with % and some option. Then following variables are used to get value which should be insert into % mark.

None of those changes with some parameter will work.
Some of those parameters also require some additional work, in "ETH.localIP" returns uint32_t or array of 4 bytes (please check what is returned here), so additional formatting to produce "192.168.1.1" string has to be done here.

Reply:
thanks for review and explanations, I will fix that :)